src/components/Loader/SpinningLoader.tsx

```diff
@@ -0,0 +1,62 @@
+import styled, { keyframes } from 'styled-components'
+
+const StyledPollingDot = styled.div`
+  width: 8px;
+  height: 8px;
+  min-height: 8px;
+  min-width: 8px;
+  border-radius: 50%;
+  position: relative;
+  background-color: ${({ theme }) => theme.surface3};
+  transition: 250ms ease background-color;
+`
+
+const StyledPolling = styled.div`
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 2px;
+  margin-left: 2px;
+  align-items: center;
+  color: ${({ theme }) => theme.neutral1};
+  transition: 250ms ease color;
+  ${({ theme }) => theme.deprecated_mediaWidth.deprecated_upToMedium`
+    display: none;
+  `}
+`
+
+const rotate360 = keyframes`
```

Comment:
i count 4 separate usages of rotate360, can we just move to const?

Reply:
pls this is copy/pasted from other file, dont make me refactor, im just a girl